src/lib/core/fiber_channel.c

```diff
@@ -41,6 +41,19 @@ enum fiber_channel_wait_status {
 	FIBER_CHANNEL_WAIT_CLOSED /* Wait is aborted, the channel is closed. */
 };
 
+/**
```

Comment:
Let's keep some meaningful order of declarations and definitions. Usually it is structures-globals-functions. I see no reason to bend the rules.

Reply:
Done.

Reply:
The static variable is still in between of types declarations.